backend/controllers/Task/DestroyController.js

```diff
@@ -0,0 +1,26 @@
+class DestroyController {
+    constructor(taskService, httpStatusCodes) {
+        this.taskService = taskService;
+        this.httpStatusCodes = httpStatusCodes;
+    }
+
+    async invoke(req, res) {
+        const { id } = req.params;
+
+        const task = await this.taskService.findById(id);
+
+        if (!task) {
+            return res.sendStatus(404);
```

Comment:
when not found in destroy then return 204

Reply:
done